src/const.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+export const SSH_KEYS_DEFAULT_TMP_FOLDER = '/tmp/tmp-ssh-keys';
+export const SSH_PUBLIC_KEY_FILENAME = 'google_compute_engine.pub';
+export const SSH_PRIVATE_KEY_FILENAME = 'google_compute_engine';
+export const GOOGLE_SSH_KEYS_TEMP_DIR_VAR = 'GOOGLE_SSH_KEYS_TEMP_DIR';
```

Comment:
nit: lets standardize on `GOOGLE_GHA_*`
```suggestion
export const GOOGLE_SSH_KEYS_TEMP_DIR_VAR = 'GOOGLE_GHA_SSH_KEYS_TEMP_DIR';
```